Makefile

```diff
@@ -37,7 +37,7 @@ help:
 	@echo "  DEBUG            -- debug flag, if any ($(DEBUG))"
 
 .PHONY: test
-test: generate unit lint dep-check
+test: generate unit lint
```

Comment:
Maybe you should also remove the other dep * targets

Reply:
Agree all dep targets should be removed as unused.

Reply:
done